### PR TITLE
Refactor views to accomodate PIWIK requirements

### PR DESCRIPTION
**Pull request by mateuszc-kainos**
## Description

Refactor views to accomodate PIWIK requirements

## Coding Standards

Code Style Guide: https://github.com/companieshouse/dissolution-web/wiki/Code-Style-Guide

## Checklist

- [ ] 3 Amigos
- [ ] Acceptance Criteria met
- [ ] Branch named {feature|hotfix|task}/{S4-*}
- [ ] Commit messages are meaningful
- [ ] Manually tested
- [ ] All unit tests passing
- [ ] WAVE accessibility testing tool ran on new or updated pages
- [ ] Pulled latest master into feature branch
- [ ] Code reviewed
- [ ] New Docker environment variables are consistent with those on the Rebel1 environment
- [ ] If your pull request depends on any other, please link them in the description

## Screenshots of new or updated views

### CODE REVIEW

File: src/views/select-director.njk [hunk 1]
```diff
@@ -5,9 +5,7 @@
 {% from "govuk/components/radios/macro.njk"        import govukRadios %}
 {% from 'govuk/components/button/macro.njk'        import govukButton %}
 
-{% block pageTitle %}
-  Which {{officerType}} are you?
-{% endblock %}
+{% set title = "Which director are you?" %}
```
Comment (mateuszc-kainos) [inline src/views/select-director.njk:8]: Static 'director', approved by Jayti

File: src/views/not-selected-signatory.njk [hunk 2]
```diff
@@ -3,9 +3,7 @@
 {% from 'govuk/components/breadcrumbs/macro.njk' import govukBreadcrumbs %}
 {% from 'govuk/components/button/macro.njk'      import govukButton %}
 
-{% block pageTitle %}
-  Error
-{% endblock %}
+{% set title = 'Email address not authorised to sign' %}
```
Comment (mateuszc-kainos) [inline src/views/not-selected-signatory.njk:6]: Title change approved by Jayti

File: src/views/layout.njk [hunk 3]
```diff
@@ -4,8 +4,6 @@
 {% from 'govuk/components/header/macro.njk'       import govukHeader %}
 {% from 'govuk/components/phase-banner/macro.njk' import govukPhaseBanner %}
 
-{% set serviceName = 'Apply to strike off and dissolve a company' %}
```
Comment (mateuszc-kainos) [inline src/views/layout.njk:7]: Moved to nunjucksLoader

File: src/views/define-signatory-info.njk [hunk 4]
```diff
@@ -6,9 +6,7 @@
 {% from "govuk/components/back-link/macro.njk"      import govukBackLink %}
 {% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
 
-{% block pageTitle %}
-  How will the {{officerType}}s be signing the application?
-{% endblock %}
+{% set title = "How will the directors be signing the application?" %}
```
Comment (mateuszc-kainos) [inline src/views/define-signatory-info.njk:9]: Static 'director', approved by Jayti

File: src/middleware/nunjucksLoader.middleware.ts [hunk 5]
```diff
@@ -50,5 +50,7 @@ export default class NunjucksLoader {
     }
 
     app.locals.piwik = this.PIWIK_CONFIG
+
+    app.locals.serviceName = 'Apply to strike off and dissolve a company'
```
Comment (peadarkelly) [inline src/middleware/nunjucksLoader.middleware.ts:54]: Maybe consider moving the text to `constants.ts`

**Review by peadarkelly — APPROVED** [review]

**Pull request merged**